charts/influxdb/templates/backup-configmap.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.backup.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "influxdb.fullname" . }}-backup
+  labels:
+    {{- include "influxdb.labels" . | nindent 4 }}
+data:
+  backup.sh: |-
+    #!/bin/bash
+
+    set -e
+
+    DATE="$(date +%Y%m%d_%H%M%S)"
+
+    for DATABASE in $(influx -username {{ .Values.setDefaultUser.user.username }} -password "${INFLUXDB_ADMIN_USER_PASSWORD}" -host {{ include "influxdb.fullname" . }}.{{ .Release.Namespace }}.svc -port {{ .Values.config.http.bind_address | default 8086 }} -execute 'SHOW DATABASES' | sed -e '1,3d'); do
```

Comment:
if you are still backing up all dbs, why just not specify any as in the original script?

Reply:
To have the backups in separate files. Makes restore easier...

Reply:
How does it make it easier? Here is an example restore job, please see the script to restore from the 1 backup folder:

https://github.com/influxdata/helm-charts/blob/master/charts/influxdb/README.md#restores

```sh
              for db in $(influx -host $INFLUXDB_HOST -execute 'SHOW DATABASES' | tail -n +5); do
                influxd restore -host $INFLUXDB_HOST:8088 -portable -db "$db" -newdb "$db"_bak /backup
              done
```

It is a little rough around the edges but what would the new restore procedure look like? Furthermore would it make restoring the entire db easier as well?